backend/uclapi/oauth/urls.py

```diff
@@ -4,7 +4,6 @@
 
 urlpatterns = [
     url(r'myapps/shibcallback$', views.myapps_shibboleth_callback),
-	url(r'myapps', views.my_apps),
     url(r'authorise/$', views.authorise),
     url(r'shibcallback', views.shibcallback),
```

Comment:
As a note for a new diff to fix the issue I described earlier, this only requires a `$`, assuming you keep using the `url()` function. If you switch to `path()`, hopefully the document I linked should describe what you should do.